fe/fe-core/src/main/java/com/starrocks/qe/scheduler/DefaultWorkerProvider.java

```diff
@@ -290,8 +301,24 @@ private static ImmutableMap<Long, ComputeNode> buildComputeNodeInfo(SystemInfoSe
                 }
                 computeNodes.put(computeNode.getId(), computeNode);
             }
-            return ImmutableMap.copyOf(computeNodes);
         }
+
+        //add BE to Resource Pool
```

Comment:
Should add BE to Resource Pool only when `computeNodes` is not empty, because the scheduler policy in the `Coordinator` will be different if there are compute nodes.